### DISCUSSION TITLE
Torchlight in Statamic

### DISCUSSION THREAD

**Discussion by ju5t** [statamic/cms#5079] [Q&A] [answered]
We'd like to use Torchlight in Statamic. Torchlight is an awesome syntax highlighter. I made a simple tag for it and it works perfectly. There is one problem though. 

The tag does an API call. Torchlight has some logic to register each tag. It will then do a single API call to convert them. This logic currently exists for Blade, but we're using Antlers (and don't want to switch). Is there a way to go over the entire view and replace all snippets in one go? I can then look at how to implement this for Torchlight.

**Comment by ryanmitchell** [comment, marked as answer]
Why not just include your torchlight call / output as a blade partial? Then you get the benefit of the blade logic but still user antlers for everything else. You can mix and match in statamic.

**Reply by ju5t** [reply]
I didn't know about that. Should've mentioned that I am new to Statamic :)

This seems to be working. That's awesome. Thanks!

**Reply by ryanmitchell** [reply]
No problem - its a pretty neat feature. Glad I could help.

**Reply by gausoft** [reply]
> I didn't know about that. Should've mentioned that I am new to Statamic :)
> 
> This seems to be working. That's awesome. Thanks!

Please can you provide an example?

**Reply by jasonvarga** [reply]
`{{ partial:whatever }}` and then have `whatever.blade.php`
